CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# # v 1.3.1 (2021-08-22)
```

Comment:
The changelog entry without date is unreleased, so you can add it there (or make it v1.2.2)